python-threatexchange/threatexchange/fb_threatexchange/api.py

```diff
@@ -21,6 +21,47 @@
 from .api_representations import ThreatPrivacyGroup
 
 
+def get_app_token(cli_option: str = None) -> str:
+    """Get the API key from a variety of fallback sources"""
```

Comment:
doc: Enumerate the fallback sources?

Reply:
I can, but seems easy to get out of date vs reading the code below